pyoes/static/scss/pyoes/_oe-header.scss

```diff
@@ -147,4 +147,42 @@ nav.top-bar {
       width: 0;
     }
   }
+}
+
+.message-banner {
+  color: $alert-color !important;
+  font-weight: bold !important;
+  padding-top: 10px !important;
+  width: 40% !important;
+  margin: 0 auto !important;
+  text-align: center !important;
+}
+
+#onderhoud-popup {
+  padding: 25px;
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  width: 50%;
+  display: none;
+  background: $white;
+  border-style: solid;
+  max-height: 50%;
+}
+#popup-sluiten {
+  float: right;
+  padding: 5px;
+}
+
+.onderhoud-banner-detail {
+  overflow-y: scroll;
```

Comment:
ik zou hier een 'auto' op  zetten ipv scroll; dan toont hij die disabled scroll bar niet in desktop modus

Reply:
Good catch, op een Mac toont die niet...

Reply:
ik bedoelde auto op de overflow-y :p

Reply:
Ja, dit was om die "sluiten..." buiten de pop-up op te lossen. Tweede commit was die overflow-y